samples/core/src/test/java/io/oasp/gastronomy/restaurant/general/common/RestTestClientBuilder.java

```diff
@@ -57,18 +102,34 @@ private String createRestServiceUrl() {
   }
 
   /**
-   * @param jacksonJsonProvider new value of {@link #getjacksonJsonProvider}.
+   * @param jacksonJsonProvider new value of {@link #jacksonJsonProvider}.
```

Comment:
Do not link private members in public JavaDoc this will not work. Try to generate JavaDoc and look at the HTML.
In Case of getter/setter pairs document the getter properly and for the sake of JavaDoc completeness add JavaDoc to setter that just links to the getter JavaDoc (@param foo the new value of {@link getFoo()}).
If you only have a setter for injection/configuration put required doc there.